src/analyses/base.ml

```diff
@@ -375,6 +375,7 @@ struct
     | `List e -> reachable_from_value ask gs st (`Address (ValueDomain.Lists.entry_rand e)) t description
     | `Struct s -> ValueDomain.Structs.fold (fun k v acc -> AD.join (reachable_from_value ask gs st v t description) acc) s empty
     | `Int _ -> empty
+    | `Thread _ -> empty (* TODO: is this right? *)
```

Comment:
I think this is the best we can do. Developers should consider it opaque anyway. If people on MacOS access some members of this struct, the code is extremely fragile and will work on MacOS only.